plugins/org.jboss.tools.websockets.ui/src/org/jboss/tools/websockets/ui/internal/ca/messages.properties

```diff
@@ -0,0 +1,4 @@
+onCloseProposalLabel=Add @OnClose method to the WebSocket endpoint
+onErrorProposalLabel=Add @OnError method to the WebSocket endpoint
+onMessageProposalLabel=Add @OnMessage method to the WebSocket endpoint
+onOpenProposalLabel=Add @OnOpen method to the WebSocket endpoint
```

Comment:
I would use another proposal labels:
- onOpen(Session session, EndpointConfig endpointConfig): void - @OnOpen method for WebSocket endpoint
- onClose(Session session, CloseReason closeReason): void - @OnClose method for WebSocket endpoint
- onMessage(String string): void - @OnMessage method for WebSocket endpoint
- onError(Session session, Throwable throwable): void - @OnError method for WebSocket endpoint

Plus, it would be nice to have an info dialog for every proposal with detailed information.